### DISCUSSION TITLE
ScriptReference/GUIUtility-keyboardControl

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#651] [Notes]
# ScriptReference/GUIUtility-keyboardControl

https://docs.unity3d.com/ScriptReference/GUIUtility-keyboardControl.html

**Comment by andreasIsCoding** [comment]
You can clear keyboard focus by setting this property to 0
`GUIUtility.keyboardControl = 0`
